packages/unlock-js/src/web3Service.ts

```diff
@@ -917,4 +917,94 @@ export default class Web3Service extends UnlockService {
     const baseTokenURI = tokenURI.substring(0, tokenURI.lastIndexOf('/') + 1)
     return baseTokenURI
   }
+
+  /**
+   * Returns the type of the input address
+   */
+  async getEthAddressType(address: string) {
```

Comment:
@SouravInsights You also need to`CHANGELOG.md` inside `unlock-js` + `package.json`

Reply:
Added necessary logs to to `CHANGELOG.md` but what about the `package.json`?